### DISCUSSION TITLE
Fail to build ipa on windows OS with AIR SDK 50.2.3.6

### DISCUSSION THREAD

**Discussion by sol-oka** [airsdk/Adobe-Runtime-Support#2892] [Q&A] [answered]
I got this error in AIR SDK 50.2.3.6, when build the ipa  on window system! How to fix it?
ld: unknown option: -platform_version
Compilation Failled while executing : ld64

**Comment by ajwfrost** [comment, marked as answer]
Hi
Please grab the updated Apple compilers from https://transfer.harman.com/link/Brh2YO2NAELPL99TZbgimo
thanks

**Reply by raresn** [reply]
@ajwfrost so we are back to being able to compile from Windows?

**Reply by sol-oka** [reply]
Problem fixed. Thank you.
